scripts/local.sh

```diff
@@ -26,14 +26,25 @@ cellbender_method="broadinstitute:cumulus:CellBender:2.3.0"
 cellranger_version="7.0.1"
```

Comment:
for line 25 can we use 
cellbender_method="cellbender/remove-background/13" 
which is basically cellbender version 3.0